### ISSUE TITLE
Setting title name to process in bash breaks plugin

### ISSUE THREAD

**Issue by mattdarveniza** [patrik-piskay/hyperterm-tabs#14]
Hyper Version: 1.1.3
hyperterm-tabs version: 1.2.1
OS: macOS 10.12.4 (bash)

Okay so this is a bit of a strange one. I've got a script in my `.bashrc` to set the tab title to the process name like so:

```bash
case "$TERM" in
xterm*|rxvt*)
    PROMPT_COMMAND='echo -ne "\033]0;${PWD##*/}\007"'
    show_command_in_title_bar()
    {
        case "$BASH_COMMAND" in
            *\033]0*)
                ;;
            *)
                echo -ne "\033]0;${BASH_COMMAND} - ${PWD##*/}\007"
                ;;
        esac
    }
    trap show_command_in_title_bar DEBUG
    ;;
*)
    ;;
esac
```

This works fine in a stock version of hyper, but since the latest version of hyper (1.3.3), installing `hyperterm-tabs` breaks completely with this code in my bashrc.

Steps to reproduce:
* Add above code to bashrc
* Install `hyperterm-tabs` and hard-reload
* First tab loaded seems to load okay and is functional, as expected
* When opening a second tab, error occurs in console, and all inputs/outputs on all tabs are frozen
* Same error occurs on opening any new tabs or even switching tabs

Error in console: `index.js:258 Uncaught TypeError: Cannot read property 'document_' of undefined`

I know this is a bit obscure as it's a bad combo of strange config conflicting with a plugin, but any ideas as to what's going on here?

**Comment by WaveHack** [comment]
Looks related to issue #9 and should've been fixed by PR #15.

Please wait until @patrik-piskay reviews and (hopefully) merges #15 and releases an update.

**Comment by patrik-piskay** [comment]
Hi @mattdarveniza, I just published v1.3.5 with the fix from @WaveHack, please let us know if it fixes your issue too!

**Comment by mattdarveniza** [comment]
Can confirm 1.3.5 resolves this issue, thanks a lot 🎉
